easybuild/easyconfigs/b/BLIS/BLIS-0.8.1-GCC-11.2.0.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'BLIS'
+version = '0.8.1'
+
+homepage = 'https://github.com/flame/blis/'
+description = """BLIS is a portable software framework for instantiating high-performance
+BLAS-like dense linear algebra libraries."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+source_urls = ['https://github.com/flame/blis/archive/']
+sources = ['%(version)s.tar.gz']
+patches = [
+    '%(name)s-%(version)s_fix_dgemm-fpe-signalling-on-broadwell.patch',
+]
+checksums = [
+    '729694128719801e82fae7b5f2489ab73e4a467f46271beff09588c9265a697b',  # 0.8.1.tar.gz
+    # BLIS-0.8.1_fix_dgemm-fpe-signalling-on-broadwell.patch
+    '345fa39933e9d1442d2eb1e4ed9129df3fe4aefecf4d104e5d4f25b3bca24d0d',
+]
+
+builddependencies = [
+    ('binutils', '2.37'),
```

Comment:
@SebastianAchilles `binutils` is already part of the `GCC` toolchain, so not needed here?